client/starwhale/api/_impl/loader.py

```diff
@@ -136,6 +136,23 @@ class DataField(t.NamedTuple):
     ext_attr: t.Dict[str, t.Any]
 
 
+class ResultLoader:
+    def __init__(
+        self,
+        datas: t.List[t.Any],
```

Comment:
`datas` is not the many data item.  use `data` directly?

Reply:
done